docs/en/base/code.md

```diff
@@ -22,11 +22,11 @@ If you want to refer to a larger piece of code, use <code>&lt;pre&gt;</code> tog
 View example of the base code block
 </a>
 
-### Copyable
+### Snippet
```

Comment:
Shouldn't this stay as Copyable?

Reply:
Oh yes, my mistake. (I initially thought that we were keeping the other one)